src/lib/draw-and-pick.js

```diff
@@ -206,11 +195,122 @@ export function pickLayers(gl, {
 }
 /* eslint-enable max-depth, max-statements */
 
+export function pickLayersByBoundingBox(gl, {
```

Comment:
@ibgreen sorry, I'm having trouble finding this in the RFC. What does it mean for an element to be "hidden"? We should try to follow the DOM model for this -- if it's "turned off" / "hidden", it's not selectable. If its opacity is 0, it is selectable. I'm not clear on exactly how this maps to deck.gl (or even if it does), nor am I clear on how we'd handle the `opacity === 0` case.